dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -77,15 +77,21 @@ internal class RumViewManagerScope(
 
     @WorkerThread
     private fun startApplicationLaunchView(event: RumRawEvent, writer: DataWriter<Any>) {
-        val applicationLaunchViewTimeNs = appStartTimeProvider.appStartTimeNs
+        val processStartTime = appStartTimeProvider.appStartTimeNs
+        // processStartTime is the time in nanoseconds since VM start. To get a timestamp, we want
+        // to convert it to milliseconds since epoch provided by System.currentTimeMillis.
+        // To do so, we take the offset of those times in the event time, which should be consistent,
+        // then add that to our processStartTime to get the correct value.
+        val timestampNs = (TimeUnit.MILLISECONDS.toNanos(event.eventTime.timestamp - event.eventTime.nanoTime)) +
```

Comment:
shouldn't it be like `(TimeUnit.MILLISECONDS.toNanos(event.eventTime.timestamp) - event.eventTime.nanoTime)` here? given that `nanoTime` is actually in nanoseconds.

Reply:
🤦‍♂️ Yes. Fixed.